public/input_text_load_data.js

```diff
@@ -119,6 +146,43 @@ function loadData(collection) {
   });
 }
 
+// Changes XML to JSON
```

Comment:
I think we need to extract the temporal values and the display property to pass along as JSON; the original XML should be held onto for editing and for selecting a display property.